mosec/manager.py

```diff
@@ -243,22 +243,22 @@ def check_and_start(self, first: bool) -> Union[WorkerRuntime, None]:
         return None
 
 
-class Controller:
+class RsRuntimeManage:
```

Comment:
Thanks for the fixing, a minor typo here.

Reply:
> Thanks for the fixing, a minor typo here.

Thanks, fixed.